utils/message_stock.py

```diff
@@ -0,0 +1,7 @@
+def message_stock(num) -> str:
```

Comment:
Again This is not how we discussed the current code should be. Inside the "actors" folder, There should be messages.py

the messages.py should have all the message classes. and should inherit from a Class called Message.
`class StartAcquisitionMessage(Message)`
`class StopAcquisitionMessage(Message)`
`class CloseApplication(Message)`